packages/graph-editor/src/editor/actions/deleteNode.tsx

```diff
@@ -1,6 +1,73 @@
-import { Dispatch } from '@/redux/store.js';
+import { Dispatch, store } from '@/redux/store.js';
 import { Graph } from '@tokens-studio/graph-engine';
 import { ReactFlowInstance } from 'reactflow';
+import type { DockLayout, TabData } from 'rc-dock';
+import type { MutableRefObject } from 'react';
+
+interface NodeWithInnerGraph {
```

Comment:
I still think that this is going to be a problem , any node that might contain more than one innergraph could cause an issue here.  It might be a better idea to transition to an explicit annotation on the node like `hasSubgraphs` where the value is a property accessor to get the graphs. 

Either that or we make an explicit function on the node that has that function